docs/dev_contribution.rst

```diff
@@ -1,50 +1,52 @@
 
 Overview
 ========
 
-For any body of work intended for the :term:`prime repository` start with a `GitHub <https://github.com/>`_ issue.
+For any body of work intended for contribution to the :term:`prime repository`, first start with a Github issue.
 The issue can be used to discuss the topic and clarify any problems related to it.
 Once progress has been made towards addressing the issue, a pull request is created that references the issue.
 
 Reviewers provide feedback on the changes by adding comments to the pull request or associated commits.
-The `Buildbot <https://buildbot.net/>`_ ``build/test`` procedure will run each time changes are pushed to the pull request's branch, and the results are displayed in the pull request view.
+The :buildbot:`Buildbot<>` :code:`build/test` procedure will run each time changes are pushed to the pull request's branch, and the results are displayed in the pull request view.
 
-Once all the changes and reviews are complete, one of the :term:`prime repository` owners will merge the pull request into the prime repository, onto the ``develop`` branch.
+Once all the changes and reviews are complete, one of the :term:`prime repository` owners will merge the pull request into the prime repository, onto the :code:`develop` branch.
 
 Note that a bug is just a type of issue, and that resolving the bug should have both the implementation to fix the bug and a test that triggers the bug.
 
 :numref:`Figure %s <libcellml-development-process>` gives a graphical overview of the developer contribution process.
 For more details, see the text below.
 
 .. _libcellml-development-process:
-.. figure:: images/libCellMLProcesses-DevelopmentProcess.png
+.. figure:: ./images/libCellMLProcesses-DevelopmentProcess.png
    :align: center
    :alt: Developer contribution process.
 
    Developer contribution process.
 
-GitHub Issue
-============
+GitHub issues
+=============
 
 If an issue does not exist for the required work (e.g. implementation of a feature, fixing of a bug), then create a new one.
 The issue is the place to discuss the particulars related to the issue, discussions on determining the scope of the issue or clarification of any points that are unclear.
 
+To create a new issue, go to the :libcellml_repo:`libCellML repository` GitHub page, and select the :code:`Issues` tab.
+Use the green button on the right hand side to create a new issue.  
+You will need to add a description of what you've encountered, as well as any small snippets of code that help describe it.
+
+
 Labels
 ------
-
-A `GitHub <https://github.com/>`_ issue may be assigned labels by the project administrators to help identify its status at a glance.
-General labels currently used for libCellML are:
+A Github issue may be assigned labels by the project administrators to help identify its status at a glance.
+General labels currently used for libCellML are: **TODO These are not current?**
 
```

Comment:
These labels don't line up with the ones we've got, which are: 
![image](https://user-images.githubusercontent.com/46887220/107565471-a5339b80-6c48-11eb-9152-989670ea1faa.png)